build/jenkins/jobs/build/build.dsl

```diff
@@ -0,0 +1,49 @@
+job('build') {
+
+    configure { project ->
+        project / 'properties' / 'hudson.plugins.copyartifact.CopyArtifactPermissionProperty' / 'projectNameList' {
+            'string' "*rook*"
+        }
+    }
+
+    parameters {
+        stringParam('branch', 'master')
+        stringParam('parallel', '')
+        stringParam('CCACHE_DIR', '')
+        stringParam('WORKSPACE', '')
+    }
+    
+    customWorkspace('${WORKSPACE}')
+
+    wrappers {
+        credentialsBinding {
+            usernamePassword('GITHUB_USERNAME', 'GITHUB_TOKEN','quantumbuild')
+
+        }
+    }
+
+    throttleConcurrentBuilds {
+        maxPerNode(1)
+        maxTotal(1)
+    }
+
+    //// not yet handling build artifacts
+    // publishers {
+    //     archiveArtifacts {
+    //         pattern('results/*')
+    //         onlyIfSuccessful()
+    //     }
+    // }
+
+    steps {
+
+        shell('git submodule sync --recursive')
+        shell('git submodule update --init --recursive')
+        shell('mkdir -p ${CCACHE_DIR}')
+        shell('(CCACHE_DIR=${CCACHE_DIR} build/run make release -j${parallel})')
```

Comment:
not sure if this is relevant, but shouldn't this be `make -j${parallel} release`. I thought make takes the options before targets. same below in numerous places.